### PR TITLE
bdd for delete recommendation

### PR DESCRIPTION
**Pull request by sadam0930**

### CODE REVIEW

File: features/recommendations.feature [hunk 1]
```diff
@@ -4,19 +4,26 @@ Feature: The recommendation service back-end
...
 	Then I should see "1"
 	And I should see "2"
 	And I should see "3"
+
+Scenario: Delete a recommendation
+    When I delete "/recommendations" with id "3"
+    And I visit "/recommendations"
+    Then I should see "1"
+    And I should see "2"
+    And I should not see "4"
```
Comment (0x932b) [inline features/recommendations.feature:29]: @sadam0930  And I should not see "3"

Comment (sadam0930) [reply features/recommendations.feature:29]: `in` is going to find 3 as the related_product_id of row 2. That's why I made it 4.

Comment (sadam0930) [reply features/recommendations.feature:29]: @u932b, does that make sense? Can this be pulled in?

Comment (0x932b) [reply features/recommendations.feature:29]: yeah we can refined later

**Review by 0x932b — APPROVED** [review]

File: features/recommendations.feature (hunk 1 above)
Comment (0x932b) [reply features/recommendations.feature:29]: I am not sure how to write bdd for a specific key, right now I am doing lazy pasting as well

**Pull request merged**